.editorconfig

```diff
@@ -0,0 +1,15 @@
+# EditorConfig helps developers define and maintain consistent
```

Comment:
Use pnpm?

Reply:
Yeah, has already set up pnpm in my local. Will push in like 20 mins